58.py

```diff
@@ -0,0 +1,15 @@
+class Solution:
+    def lengthOfLastWord(self, s: str) -> int:
+        word_starts = False
+        word_length = 0
+
+        for char in reversed(s):
```

Comment:
Instead of reversing `s`, why can't we just go through each character from the end of the string all the way back to its beginning?